doc: nrf: Update Monitor mode documentation (#14038)
nrfconnect/sdk-nrf: doc/nrf/protocols/wifi/sniffer_rx_operation.rst

   IEEE 802.11 packet sniffer operational flow

.. _ug_nrf70_developing_monitor_mode_receive_operation_filter_setting:

Monitor mode receive operation filter configuration
***************************************************

> The type of IEEE 802.11 receive packets that the application receives can be configured using the filter ``NET_REQUEST_WIFI_PACKET_FILTER`` net management command.

Reviewer: ```suggestion
The type of IEEE 802.11 packets that the application receives can be configured using the filter ``NET_REQUEST_WIFI_PACKET_FILTER`` Wi-Fi management command.
```

> The nRF70 Series Wi-Fi driver has to be configured to operate in Monitor mode or Promiscuous mode for setting packet filter settings.

Reviewer 2: ```suggestion
The nRF70 Series Wi-Fi driver must be configured to operate in Monitor mode or Promiscuous mode for setting packet filter settings.
```

> The application can choose to receive 802.11 management packets, 802.11 control packets, 802.11 data packets or a combination of the three types of 802.11 packets.

Reviewer 2: ```suggestion
The application can choose to receive 802.11 management packets, 802.11 control packets, 802.11 data packets, or a combination of the three types of 802.11 packets.
```
